Google.PowerShell/Provider/GoogleCloudStorageProvider.cs

```diff
@@ -522,6 +528,10 @@ protected override void GetChildItems(string path, bool recurse)
         /// <param name="newItemValue">The value of the item to create. We assume it is a string.</param>
         protected override void NewItem(string path, string itemTypeName, object newItemValue)
         {
+            if (!ShouldProcess(path, "New-Item"))
```

Comment:
As it is written, the output is this:

    PS gs:\jimwp-test-bucket\> new-item x -whatif
    What if: Performing the operation "New-Item" on target "jimwp-test-bucket\x".